examples/aa.dawg

```diff
@@ -0,0 +1,50 @@
+# Test aa evaluationary models
+
+Tree.Tree = "(A:0.3, B:0.2)C;"
+Tree.Scale = 0.99999
+Root.Length = 100
+Root.Code = 2
+Root.Seq = "AATGCTTTTTTTTTTGGCCAA"
+Sim.Reps = 10
+Sim.Seed = 42
+Root.Seg = 0
+
+Subst.Model = "aagtr"
+Subst.Freqs  = 0.2, 0.3, 0.3, 0.2
+Subst.Params = 2.0, 1.0, 3.0, 1.0, 1.0, 1.0
+Subst.Rate.Model = "ZERO"
+
+Subst.Model = "equ"
```

Comment:
You are overwriting the previous model here. Did you realize that?

Reply:
Should they be part of separate segments?

Reply:
Separate segments or separate files.

Reply:
I'm having trouble setting up a `aagtr` model. It says it needs 20 frequencies and 190 parameters, I'm trying to use this:
```python
[[-]]
Subst.Model = "aagtr"
Subst.Freqs  = 0.2, 0.3, 0.3, 0.2, 0.2, 0.3, 0.3, 0.2, 0.2, 0.3, 0.3, 0.2, 0.2, 0.3, 0.3, 0.2, 0.2, 0.3, 0.3, 0.2
Subst.Params =
2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0,
2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0,
2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0,
2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0,
2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0,
2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0,
2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0,
2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0, 1.0, 1.0, 2.0, 1.0, 3.0, 1.0
Subst.Rate.Model = "ZERO"
```